LoRaEngine/modules/LoRaWanNetworkSrvModule/Dockerfile.amd64

```diff
@@ -8,6 +8,11 @@ COPY global.json ./
 COPY LICENSE ./
 COPY ./LoRaEngine/modules/LoRaWanNetworkSrvModule/NOTICE.txt ./
 
+WORKDIR /build/LoRaEngine/modules
```

Comment:
Why is this needed? Isn't the dotnet restore executed anyways from the LoRaWanNetworkSrvModule as the Discovery project is being referenced in its csproj?

Reply:
Ditto for other dockerfiles

Reply:
I would still need to switch back to copy over the discovery service files after the restore has executed, no? I would be able to get rid of one restore, but would need another workdir switch instead. Or am I missing something?

Reply:
Sorry, I was referring explicitly to the dotnet restore. I would avoid operation on rows 12/13 and just switch workdir and then "COPY ./LoRaEngine/modules/LoRaWan.NetworkServerDiscovery LoRaWan.NetworkServerDiscovery"

Reply:
Addressed in 1cd6ab6fe18b6fe55c1f325e37c75f946b0bb6db